fp.js

```diff
@@ -31,4 +32,8 @@ var people = [
   {name: "Jessica", age: 34},
   {name: "Jasmine", age: 39}
 ]
-var peopleWhoseNamesHaveTheLetterS = people.filter(person => person.name.match("s"))
+peopleWhoseNamesHaveTheLetterS = []
+people.forEach(person => {
+  peopleWhoseNamesHaveTheLetterS.push(person.name.match("s"))
+})
```

Comment:
close, need a little bit of logic to check to see if that person's name contains the little s, before pushing. i.e:

```js
peopleWhoseNamesHaveTheLetterS = []
 people.forEach(person => {
   if ( person.name.match("s") ) {
      peopleWhoseNamesHaveTheLetterS.push(person)
  }
 })
```